src/app/portlets/dot-edit-content/services/html/dot-edit-content-toolbar-html.service.ts

```diff
@@ -130,4 +109,15 @@ export class DotEditContentToolbarHtmlService {
             });
         });
     }
+
+    private getContentButton(identifier: string, inode: string, dragLabel: string, editLabel: string, removeLabel: string): string {
```

Comment:
Way to many params, is a good practice in JS that when you have more than 2 params use an object instead.